### DISCUSSION TITLE
Can't input chinese characters in input component

### DISCUSSION THREAD

**Discussion by DDowl51** [ArthurSonzogni/FTXUI#372] [Q&A]
I thought it was encode problem but displaying chinese characters is normal.
and I tried this example: https://arthursonzogni.com/FTXUI/examples/?file=component/input
the result was the same
Is it a bug or just can't support chines
![pic](https://user-images.githubusercontent.com/53144603/161508078-3d7ef934-f4de-4724-a2b3-1bde962d2bda.png)
e

**Comment by DDowl51** [comment]
and I found chinese support normal under linux environment

**Comment by ArthurSonzogni** [comment]
Other users reported this was working correctly. See: https://github.com/ArthurSonzogni/FTXUI/issues/2
I was told most of CJK user use IME ([Input Method](https://en.wikipedia.org/wiki/Input_method)). Are you using this to input chinese UTF8 characters?

**Comment by DDowl51** [comment]
Yes im using IME to input
and i use this example: https://arthursonzogni.com/FTXUI/examples/?file=component/print_key_press
to test cjk characters input on both windows10 and ubuntu
the results are
![resultsonwindows](https://user-images.githubusercontent.com/53144603/162583262-11822300-cf45-4c1a-a3ae-59d27c45634c.png)
![resultonubuntu](https://user-images.githubusercontent.com/53144603/162583270-f5f40d9c-511d-4f36-bcb7-2b5c64563c6b.png)

**Reply by ArthurSonzogni** [reply]
Thanks for the screenshots!
I don't have a Microsoft OS and don't know how to setup IME.
If someone passing by have any clue, please let us know.

**Reply by DDowl51** [reply]
maybe it's problem about Windows system 
when you input chinese characters with IME, it automatically encode it with GBK(Chinese Internal Code Specification) even if you set code page to Unicode, cant find a solution about this
Thanks anyway, it's a cool library!
